contracts/PWNDeed.sol

```diff
@@ -23,24 +23,29 @@ contract PWNDeed is ERC1155, Ownable {
      */
     uint256 public id;
 
+    /**
+     * EIP-1271 valid signature magic value
+     */
+    bytes4 constant internal EIP1271_VALID_SIGNATURE = 0x1626ba7e;
```

Comment:
This seems like a requirement of EIP 1271?  I'd just kinda like to understand this value more.

Reply:
Yes. You can read more about the EIP here https://eips.ethereum.org/EIPS/eip-1271 . It's not that long and you can fully understand it. There is also an example which could clarify the usage.